packages/ilios-common/addon/components/course/rollover-date-picker.hbs

```diff
@@ -1,8 +1,6 @@
 <input
   aria-label={{t "general.pickADate"}}
   data-test-course-rollover-picker
-  {{did-insert (perform this.showPicker) @course @value}}
-  {{did-update (perform this.showPicker) @course @value}}
-  {{on "click" (if this.isOpen this.close (noop))}}
+  {{get-element (perform this.showPicker @course @value)}}
```

Comment:
I've been leaving these with a `did-insert` and figured we'd clean them up. I have no problem with this approach though, let's do it! We have a few places where we *do* need the element for something and this is a good solution.